Code/Tools/AssetProcessor/native/ui/SourceAssetTreeModel.h

```diff
@@ -45,5 +52,7 @@ namespace AssetProcessor
         AZStd::unordered_map<AZ::s64, AssetTreeItem*> m_sourceIdToTreeItem;
         QDir m_assetRoot;
         bool m_assetRootSet = false;
+        bool m_intermediateAssets = false;
```

Comment:
You have Ctor code for this class, considering setting values in initializer list in single place. As is I have to look at both the .cpp and the .h to discover default values

Reply:
The more you know! I learnt something here, my preference is to only initialize in once place, but I see I have to adjust my preference now ;)